addon/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/with-contenv bashio
+
+CONFIG_PATH=/data/options.json
+
+export mqtt_server="$(bashio::config 'mqtt_server')"
+export mqtt_user="$(bashio::config 'mqtt_user')"
+export mqtt_password="$(bashio::config 'mqtt_password')"
+export debug="$(bashio::config 'debug')"
+
+if [[ $debug == "true" ]]
+then
+	echo Debug mode on - sleeping indefinitely
+	sleep inf
+fi
+
+#Test to see if we can connect to the Outlander
+nc -zw 1 192.168.8.46 8080
+RC=$?
+
+if [[ $RC -ne  0 ]]
+then
+	echo Connection to Outlander unsuccessful
```

Comment:
This means that the addon won't start if the outlander is not nearby? Would it be better just to proceed anyway...phev2mqtt will retry indefinitely.
Perhaps just checking wifi is available or something? Although there can also be the possibility that the wifi is connected on some other device (eg I currently use a separate device nearer the car to get wifi reachability).

Reply:
Yes, initially I was checking that the wireless lan was connected to the right subnet but then thought that would be no good if it wasn't connected directly.
I was going to leave the check in (as I suspect the first time someone's trying this the Outlander will be in range), so the info that it couldn't connect might be useful but remove the "exit 1" but the failure of the netcat causes the script to exit anyway, so I'll remove that whole check.